### DISCUSSION TITLE
add unicode feature

### DISCUSSION THREAD

**Discussion by movingsnail007** [outline/outline#3655] [Ideas]
It seems unicodes like \&deg; will not work in outlinewiki.
Unicodes are sometimes very necessary for documentation.

**Comment by tommoor** [comment]
I'm not sure what you mean here, Outline doesn't support this html specific encoding (or any html) but you can just use the character directly, eg °c

**Reply by movingsnail007** [reply]
Thanks tommoor, for the quick response, and sorry for the confusion created.
as far as I know, for those special characters, I can only copy and paste them into Outline page body.
In Github, we can just type in \&deg;c and it will be translated into &deg;c.
It will be very convenient, at least in my opinion to type those special characters with HTML-specific encoding.
Sure, we will need an escape character when we don't want to use the encoding.
